vision/style_transfer/README.md

```diff
@@ -0,0 +1,82 @@
+# Fast Neural Style
+
+## Use-cases
+This artistic style transfer model mixes the content of an image with the style of another image. Below are examples of style transfer models. Each model transfers a unique style to a photograph of a door arch.  
+
+<p align="center">
+  <img src="images/input_images/amber.jpg" width="50%" height="50%">
+</p>
+
+|Model        |Mosaic  |Candy  | Rain Princess |Udnie  |Pointilism|
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
+|Style Images       |[<img src="images/style_images/mosaic.jpg">](images/style_images/mosaic.jpg)| [<img src="images/style_images/candy.jpg">](images/style_images/candy.jpg)|[<img src="images/style_images/rain_princess.jpg">](images/style_images/rain_princess.jpg)| [<img src="images/style_images/udnie.jpg">](images/style_images/udnie.jpg)| [<img src="images/style_images/pointilism.jpg">](images/style_images/pointilism.jpg)|
```

Comment:
Could you confirm that we have permission/license to use these images?

Reply:
Instead of duplicating the images here, can we just to https://github.com/pytorch/examples/tree/master/fast_neural_style#models which shows the examples anyways?

"This artistic style transfer model mixes the content of an image with the style of another image. Examples of these can be seen [here](https://github.com/pytorch/examples/tree/master/fast_neural_style#models)"

Reply:
@prasanthpul Thanks for the feedback. Do you think I should also get rid of the "images" folder (which contains input_images, output_images and style_images)?

Reply:
If they are not referenced from anywhere else, then yes you should delete them.